### PR TITLE
change plot model to fully support plotting submodel

### PR DESCRIPTION
**Pull request by ccgcccccc**
there are some explanations

### CODE REVIEW

File: keras/utils/vis_utils.py [hunk 1]
```diff
@@ -61,7 +63,7 @@ def model_to_dot(model,
 
     _check_pydot()
     if subgraph:
-        dot = pydot.Cluster(style='dashed')
+        dot = pydot.Cluster(style='dashed', graph_name=model.name)
```
Comment (ccgcccccc) [inline keras/utils/vis_utils.py]: this is to prevent different submodel confusing

File: keras/utils/vis_utils.py [hunk 2]
```diff
@@ -38,6 +39,7 @@ def model_to_dot(model,
                  expand_nested=False,
                  dpi=96,
                  subgraph=False):
+                 #subgraph=True):
```
Comment (ccgccc) [inline keras/utils/vis_utils.py]:
```suggestion
```
please remove unnessary comments

File: keras/utils/vis_utils.py [hunk 3]
```diff
@@ -100,6 +104,33 @@ def model_to_dot(model,
                 layer_name = '{}({})'.format(layer_name, layer.layer.name)
                 child_class_name = layer.layer.__class__.__name__
                 class_name = '{}({})'.format(class_name, child_class_name)
+        '''
+        # ccg modify - original end
+
```
Comment (ccgccc) [inline keras/utils/vis_utils.py:86]: please remove

File: test_file_created_when_merging.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+test the influence of adding a unrelevant file when merging.
```
Comment (ccgccc) [inline test_file_created_when_merging.txt]: please remove this file

**Review by ccgccc — CHANGES_REQUESTED** [review]

**Review by ccgccc — APPROVED** [review]

**Comment by ccgccc** [comment]
Please make sure your codes meet PEP8 standard.

**Comment by ccgccc** [comment]
please create another PR again.

**Pull request closed without merging**